clang/include/clang/Basic/Attr.td

```diff
@@ -3220,8 +3220,8 @@ def TypeVisibility : InheritableAttr {
   let Args = [EnumArgument<"Visibility", "VisibilityType",
                            ["default", "hidden", "internal", "protected"],
                            ["Default", "Hidden", "Hidden", "Protected"]>];
-//  let Subjects = [Tag, ObjCInterface, Namespace];
-  let Documentation = [Undocumented];
+  let Subjects = SubjectList<[Tag, ObjCInterface, Namespace], ErrorDiag>;
```

Comment:
I'm not certain if this is correct, and whether this means I need to add some tests.

Reply:
I would separate out this change because it's not really related to documenting the attribute (this is used to generate automatic checks for the subject, so it corresponds to this code that was manually written and can possibly be removed: https://github.com/llvm/llvm-project/blob/091fc81d485594d4b751c10518230769c905430c/clang/lib/Sema/SemaDeclAttr.cpp#L2917)